dali/test/python/nose_utils.py

```diff
@@ -12,6 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import nose
```

Comment:
Do we need this in this PR?

Reply:
Not really, I can remove.

Reply:
I would move it to a separate one if you want ot merge it.

Reply:
Done.